src/services/shims.ts

```diff
@@ -247,14 +247,14 @@ namespace ts {
         }
 
         public getChangeRange(oldSnapshot: IScriptSnapshot): TextChangeRange {
-            var oldSnapshotShim = <ScriptSnapshotShimAdapter>oldSnapshot;
-            var encoded = this.scriptSnapshotShim.getChangeRange(oldSnapshotShim.scriptSnapshotShim);
+            let oldSnapshotShim = <ScriptSnapshotShimAdapter>oldSnapshot;

```

Comment:
These should change to const instead of let. This applies to a lot of the other edits too. Is this file not checked by tslint?